apps/settings/js/panels/screen_lock_passcode/screen_lock_passcode.js

```diff
@@ -1,9 +1,11 @@
-/* global SettingsListener */
+/* global PasscodeHelper */
 define(function(require) {
   'use strict';
 
   var SettingsService = require('modules/settings_service');
   var SettingsUtils = require('modules/settings_utils');
+  require('/shared/js/passcode_helper.js');
```

Comment:
To require a shared module using requirejs, we need to declare a shim here[1].

[1]: https://github.com/mozilla-b2g/gaia/blob/master/apps/settings/js/config/require.js#L18

Then we can use `var PasscodeHelper = require('shared/passcode_helper');`.